### PR TITLE
[Enhancement] Adding exclusions for libraries to fix CVEs

### PR DESCRIPTION
**Pull request by va-os-commits**
## Why I'm doing:
This is my first PR for Fixing VULNS for libraries used in the repo
https://snyk.io/vuln/SNYK-JAVA-ORGCODEHAUSJACKSON-3326362
https://snyk.io/vuln/SNYK-JAVA-ORGJSON-5488379
https://snyk.io/vuln/SNYK-JAVA-ORGECLIPSEJETTYHTTP2-5958918

## What I'm doing:
Excluding libraries with VULNS

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.3
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: fe/fe-core/pom.xml
Comment (Smith-Cruise) [inline fe/fe-core/pom.xml:984]: You can't remove it, it will affect ranger work
https://github.com/StarRocks/starrocks/pull/47199

File: fs_brokers/apache_hdfs_broker/src/hadoop-cos-shaded/pom.xml
Comment (Smith-Cruise) [inline fs_brokers/apache_hdfs_broker/src/hadoop-cos-shaded/pom.xml:47]: I think you can bump it to `3.3.3-8.3.10` first, it maybe fixed cve.

Comment (va-os-commits) [reply fs_brokers/apache_hdfs_broker/src/hadoop-cos-shaded/pom.xml:47]: Let me check that

File: fe/fe-core/pom.xml
Comment (va-os-commits) [reply fe/fe-core/pom.xml:984]: Looks like VULN might already be fixed in 2.5.0 of _ranger-plugins-common_ so will revert the change

File: fs_brokers/apache_hdfs_broker/src/hadoop-cos-shaded/pom.xml
Comment (va-os-commits) [reply fs_brokers/apache_hdfs_broker/src/hadoop-cos-shaded/pom.xml]: @Smith-Cruise could you please review this again?

**Review by Smith-Cruise — APPROVED** [review]

**Comment by Smith-Cruise** [comment]
@wyb PTAL also

**Comment by va-os-commits** [comment]
> @wyb PTAL also

@wyb can you please review this when you get a chance?

**Review by wyb — DISMISSED** [review]

**Comment by Smith-Cruise** [comment]
Can you rebase the code and push it to trigger CI again?

**Comment by va-os-commits** [comment]
> Can you rebase the code and push it to trigger CI again?
@Smith-Cruise I just did that and it requires a lot more approval now. Did I mess something up?

**Review by stephen-shelby — APPROVED** [review]

**Comment by Smith-Cruise** [comment]
Fixed, but you have wait for some time, our CI service is down.....

**Review by wyb — APPROVED** [review]

**Pull request merged**
